apps/docs/styles/globals.css

```diff
@@ -1305,6 +1343,11 @@ body {
 		width: 100%;
 		justify-content: flex-end;
 	}
+
+	.article__embed--quickstart {
+		aspect-ratio: 16/9;
+		height: 280px;
```

Comment:
this should be a `min-height` to be more flexible - i converted it above for the desktop version